src/transport/SecurePairingSession.cpp

```diff
@@ -179,12 +178,6 @@ CHIP_ERROR SecurePairingSession::Init(Optional<NodeId> myNodeId, uint16_t myKeyI
 
     VerifyOrExit(delegate != nullptr, err = CHIP_ERROR_INVALID_ARGUMENT);
 
-    err = mCommissioningHash.Begin();
```

Comment:
Context is now Hash("CHIP PAKE V1 Commissioning" || PBKDFParamRequest || PBKDFParamResponse). We'll still need this hash so we don't need to save PBKDFParamRequest and PBKDFParamResponse in memory while building a secure session.